ea_airflow_util/dags/aws_param_store_to_airflow_dag.py

```diff
@@ -55,65 +63,40 @@ def insert_all_aws_params_to_airflow():
     # stackoverflow link:
     # https://stackoverflow.com/questions/51863881/is-there-a-way-to-create-modify-connections-through-airflow-api
     @staticmethod
-    def create_conn(
-            conn_id  : str,
-            conn_type: str,
-            host     : str,
-            schema   : str,
-            login    : str,
-            password : str,
-            port     : str,
-            extra    : str
-    ) -> Connection:
+    def create_conn(conn_id: str, **kwargs) -> Connection:
```

Comment:
The explicit arguments of this method have been replaced with kwargs for flexibility.